sopel/modules/help.py

```diff
@@ -82,23 +96,64 @@ def help(bot, trigger):
 
 def create_list(bot, msg):
     msg = 'Command listing for {}@{}\n\n'.format(bot.nick, bot.config.core.host) + msg
-    payload = {"content": msg}
-    headers = {'Content-type': 'application/json', 'Accept': 'application/json'}
 
     try:
-        result = requests.post('https://ptpb.pw/', json=payload, headers=headers)
-    except requests.RequestException:
+        result = globals()['post_to_' + bot.config.help.output](bot, msg)
```

Comment:
Using `globals()` sure is convenient, but that doesn't make me very comfortable: you never know what is in `globals()` and what can be exploited with that. Plus, the whole `'post_to_' + ...` is less readable than, say, this:

```
backends = {
    'ptpb': post_to_ptpb,
    '0x0': post_to_0x0,
    # and so on
}
try:
    result = backends[bot.config.help.output](bot, msg)
except KeyError:
    bot.say('Sorry! Wrong help configuration')
```